app/src/main/res/layout/fragment_maps.xml

```diff
@@ -3,11 +3,17 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     tools:context=".presentation.ui.MapsFragment">
 
-    <androidx.fragment.app.FragmentContainerView
-        android:id="@+id/map"
+    <com.google.android.gms.maps.MapView
+        android:id="@+id/map_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginTop="55dp"
+        android:layout_marginBottom="55dp"
```

Comment:
format

Reply:
done

Reply:
Margin doesn't work with style. Is there an other way to make maps fragment being under main fragment toolbar without using elevation due to SDK restriction (min API 19, doesn't support elevation)

Reply:
I'm wrong, style work well.

Reply:
Is there an other way to make toolbar beeing over maps fragment without using elevation due to SDK restriction (API 19 doesn't suport elevation)

Reply:
The fragment is supposed to display below the toolbar. 
I don't understand your question. 
Please attach a screen capture

Reply:
![Screenshot_2020-07-16-23-51-20-015_com picone go4lunch](https://user-images.githubusercontent.com/60393680/87726636-af2f1200-c7bf-11ea-9bca-6298fdfb2336.jpg)

without padding on mapsWidget

Reply:
![Screenshot_2020-07-16-23-52-14-655_com picone go4lunch](https://user-images.githubusercontent.com/60393680/87726683-cb32b380-c7bf-11ea-897a-557bc8550827.jpg)
with paddin 55dp on mapsWidget

Reply:
perhaps caused by using widget instead of fragment?